src/main/flight/alt_hold.c

```diff
@@ -131,43 +131,42 @@ float altitudePidCalculate(void)
     return output; // motor units, eg 100 means 10% of available throttle 
 }
 
-void simplePidInit(float kp, float ki, float kd, float kf)
+void altHoldPidInit(void)
+{
+    altitudePidCoeffs_t data;
+    getAltitudePidCoeffs(&data);
+    altHoldPid.kp = data.kp;
+    altHoldPid.ki = data.ki;
+    altHoldPid.kd = data.kd;
+    altHoldPid.kf = data.kf;
+    altHoldPid.integral = 0.0f;
```

Comment:
```suggestion
    altHoldPidCoeffs = *getAltitudePidCoeffs();
    integral = 0.0f;
```

Reply:
implemented, though a little differently

Reply:
unfortunately not. The way you did it is impractical because
- It takes away the possibility for direct access to the pid coefficients (we can't use efficiency when possible)
- Whenever you make a change to the struct, you also need to make a manual change to the `getAltitudePidCoeffs()` implementation. Prone to bugs and tedious work.

Reply:
OK, let's improve it! :-)

Reply:
All suggested changes implemented, tested and works properly